Detections/ASimWebSession/SuspiciousUACryptoMiners.yaml

```diff
@@ -0,0 +1,40 @@
+id: 8cbc3215-fa58-4bd6-aaaa-f0029c351730
+name: Suspicious User-Agent Detected - CryptoMiner
```

Comment:
Name: A host is potentially running a crypto miner (ASIM Web Session schema)

Description:
This rule identifies a web request with a user agent header known to belong to a crypto miner. This indicates a crypto miner may have infected the client machine.\<br\>\<br\>This rule uses the Advanced SIEM Information Model (ASIM) and supports any web session source that complies with ASIM. To use this Analytics Rule, [deploy the Advanced SIEM information Model (ASIM)](https://aka.ms/DeployASIM\).

alertDisplayNameFormat: The host {{SrcIpAddr}} is potentially running a crypto miner

alertDescriptionFormat:
The host at address {{SrcIpAddr}} sent an HTTP request to the URL {{Url}} with the HTTP user agent header {{HttpUserAgent}}. This user agent is known to be used by crypto miners and indicates crypto mining activity on the client. 

Entities: URL, Username if available.

Custom fields: User agent.